xOffset: 0.6
yOffset: 0.4
zOffset: 0.0
